auditlog_tests/tests.py

```diff
@@ -920,3 +923,31 @@ def test_no_delete_related(self):
             list(entries.values_list("action", flat=True)),
             [LogEntry.Action.CREATE, LogEntry.Action.UPDATE, LogEntry.Action.DELETE],
         )
+
+
+class ModelFromDifferentDatabase(TestCase):
```

Comment:
I think we don't need a new class, you can add this test to https://github.com/jazzband/django-auditlog/blob/665217d32f5d07a774fe53cafde6aef36bb4bff5/auditlog_tests/tests.py#L36